add-ons/wesnoth-The_Earths_Gut.po

```diff
@@ -74,11 +67,11 @@ msgid ""
 "separate campaign by Ashmyr/Myrian)</small>."
 msgstr ""
 " nebo vyšší.\n"
-"Vyžaduje instalaci kampaně „Thursaganovo kladivo“ (která je nyní součástí "
-"hlavní instalace, takže by ve hře měla být obsažena, pokud nebyla ručně "
-"smazána nebo tak něco).\n"
-"Bývalý pracovní název: „Trpasličí království“ (což je nyní název "
-"samostatného tažení od Ashmyr/Myrian)."
+"Vyžaduje instalaci kampaně „Thursaganovo kladivo“ <small>(která je nyní "
+"součástí hlavní instalace, takže by ve hře měla být obsažena, pokud nebyla "
+"ručně smazána nebo tak něco)</small>.\n"
+"Bývalý pracovní název: „Trpasličí království“ <small>(což je nyní název "
+"samostatného tažení od Ashmyr/Myrian)</small>."
```

Comment:
oh right, I'm going to have to update the "Dwarven Kingdom" reference, now that Myrian has renamed her campaign by that title...